docs/run/advanced-runtime-options.mdx

```diff
@@ -43,7 +47,7 @@ When creating an instance of the `Estimator` or `Sampler` class, you can pass in
 
 ```python
 estimator = Estimator(session=backend, options=options)
-result = estimator.run(circuit, observable).result()
+result = estimator.run(isa_circuit, isa_observable).result()
```

Comment:
For these examples where `circuit` and `observable` are not actually defined in the text, does it really clarify the example to rename these?

Reply:
I'm going to drop these changes.